spec/requests/id_card_attributes_request_spec.rb

```diff
@@ -6,43 +6,90 @@
   let(:token) { 'fa0f28d6-224a-4015-a3b0-81e77de269f2' }
   let(:auth_header) { { 'Authorization' => "Token token=#{token}" } }
   let(:current_user) { build(:user, :loa3) }
-  let(:service_episodes) { [build(:service_episode)] }
 
   before do
     allow(Settings.vic).to receive(:signing_key_path)
       .and_return(::Rails.root.join(*'/spec/support/certificates/vic-signing-key.pem'.split('/')).to_s)
+
     sign_in_as(current_user)
   end
 
   describe '#show /v0/id_card/attributes' do
-    it 'returns a signed redirect URL' do
-      expect_any_instance_of(EMISRedis::MilitaryInformation)
-        .to receive(:service_episodes_by_date).at_least(:once).and_return(service_episodes)
-      expect_any_instance_of(VAProfile::VeteranStatus::Service)
-        .to receive(:title38_status).at_least(:once).and_return('V1')
-      get '/v0/id_card/attributes', headers: auth_header
-      expect(response).to have_http_status(:ok)
-      json = JSON.parse(response.body)
-      url = json['url']
-      expect(url).to be_truthy
-      traits = json['traits']
-      expect(traits).to be_key('edipi')
-      expect(traits).to be_key('firstname')
-      expect(traits).to be_key('lastname')
-      expect(traits).to be_key('title38status')
-      expect(traits).to be_key('branchofservice')
-      expect(traits).to be_key('dischargetype')
-      expect(traits).to be_key('timestamp')
-      expect(traits).to be_key('signature')
+    context 'Feature flipper military_information_vaprofile=false' do
+      let(:service_episodes) { [build(:service_episode)] }
+
+      before do
+        Flipper.disable(:military_information_vaprofile)
```

Comment:
@bosawt We initially used `allow(Flipper).to receive(:enabled?).with(:military_information_vaprofile).and_return(false)` here, but we were getting odd errors that were resolved when we explicitly enabled and disabled the flipper in the test, so that's why we're doing that here. This was where the error was coming from:
```
[4] pry(#<RSpec::ExampleGroups::RequestingIDCardAttributes::ShowV0IdCardAttributes::FeatureFlipperMilitaryInformationVaprofileFalse>)> Flipper.enabled?(:veteran_status_updated)
RSpec::Mocks::MockExpectationError: Flipper received :enabled? with unexpected arguments
  expected: (:military_information_vaprofile)
       got: (:veteran_status_updated)
Diff:
@@ -1 +1 @@
-[:military_information_vaprofile]
+[:veteran_status_updated]

 Please stub a default value first if message might be received with other args as well. 

from /Users/rebeccatolmach/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/rspec-support-3.12.1/lib/rspec/support.rb:108:in `block in <module:Support>'
```
We couldn't figure out why the flipper `veteran_status_updated` was getting called in our test 😕